src/adapter/src/catalog/builtin_table_updates.rs

```diff
@@ -476,63 +476,20 @@ impl CatalogState {
         let owner_id = entry.owner_id();
         let privileges_row = self.pack_privilege_array_row(entry.privileges());
         let privileges = privileges_row.unpack_first();
-        let mut updates =
-            match entry.item() {
```

Comment:
this diff is big since it the indentation wasn't correctly formatted and now it's outdented the whole match statement, but the only update I made was within the `CatalogItem::Table(table)` case of the match

Reply:
Suppressing whitespace works perfectly here (https://github.com/MaterializeInc/materialize/pull/29374/files?w=1)—thanks for the callout!